__tests__/app.test.js

```diff
@@ -0,0 +1,39 @@
+const request = require("supertest");
+const db = require("../db/connection");
+const app = require("../app.js");
+const testData = require('../db/data/test-data/index')
+const seed = require("../db/seeds/seed")
+
+beforeEach(() => seed(testData));
+afterAll(() => {
+  db.end()
+});
+
+describe("GET/api/categories", () => {
+    test('status:200, responds with an array of categories objects', () => {
+      return request(app)
+        .get('/api/categories')
+        .expect(200)
+        .then((response) => {
+          const  categories  = response.body.categories;
+          expect(categories).toBeInstanceOf(Array);
+          expect(categories).toHaveLength(4);
```

Comment:
The length assertion is really important when using an expect statement inside a forEach, good work 👍